staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/validator.go

```diff
@@ -80,7 +81,10 @@ func (v *validator) Validate(ctx context.Context, matchedResource schema.GroupVe
 	} else {
 		f = *v.failPolicy
 	}
-
+	// if the type is explicitly excluded for security reasons, skip validation.
+	if exclusion.Excluded.Has(matchedResource) {
```

Comment:
This should be a member field that is injected or obtained at admission construction time. One possibility is the Wants____ pattern admission initializers can provide, like with an `ExcludedAdmissionResources()` function.

Reply:
Thank you. That's a good point. I am currently working to refactor in this direction.

Reply:
I've expended this idea a bit to DI an entire ResourceFilter that takes the whole `Attributes`, because I predict we may have more complicated rules, especially for mutating. Please let me know how you like this idea. Thank you.